### PR TITLE
Update Discord installation guide and mention Mattermost bot name configuration

### PR DESCRIPTION
**Pull request by pkosiec**
##### SUMMARY

- Make Discord installation guide up-to-date

    I went through the Discord tutorial and stumbled upon a few outdated pieces of information. I updated the guide (including some screenshots).

- Mention ability to configure custom bot name for Mattermost

   When I was following the guide, I named my bot name differently by accident and needed a way to configure the username. I found such possibility from GitHub issues, and I agree with author https://github.com/infracloudio/botkube/issues/569, it should be mentioned here on the documentation website.

Fixes https://github.com/infracloudio/botkube/issues/569

### CODE REVIEW

File: content/installation/Discord/_index.md [hunk 1]
```diff
@@ -47,33 +47,30 @@ Follow the steps below to install BotKube Discord app to your Discord server.
...
-    ```yaml
-      # Settings for Discord
-      discord:
-        enabled: true
-        token: 'DISCORD_TOKEN'	    # BotKube Bot Token 
-        botid: 'DISCORD_BOT_ID'         # BotKube Application Client ID 
-        channel: 'DISCORD_CHANNEL_ID'   # Discord Channel id for receiving BotKube alerts 
-        notiftype: short                # Change notification type short/long you want to receive. notiftype is optional and Default notification type is short (if not specified)
-    ```
+8. Switch to Discord app. Navigate to **User settings** and select **Advanced** tab.
```
Comment (PrasadG193) [inline content/installation/Discord/_index.md:59]:
```suggestion
8. Switch to the Discord app. Navigate to **User settings** and select **Advanced** tab.
```

**Review by PrasadG193 — APPROVED** [review]
Left minor comment as per the grammar checks

**Comment by muffl0n** [comment]
Thank you! ❤️

**Comment by pkosiec** [comment]
No problem at all @muffl0n! Cheers!

**Pull request merged**